### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by lesyeuxdechris**
Changed a few spelling things and punctuation mistakes. Edited some sentence schemes to make it sound more natural.
Twitter: rubyjanechris

### CODE REVIEW

File: _locales/pl/messages.json [hunk 1]
```diff
@@ -128,9 +128,9 @@
     "copy_profile_link": { "message": "Kopiuj link do profilu" },
     "remove_from_followers": { "message": "Usuń z obserwujących", "description": "This button unfollows person from you, not unfollows you from them" },
     "remove_from_followers_sure": { "message": "Czy na pewno chcesz usunąć @$SCREEN_NAME$ z twojej listy obserwujących?", "placeholders": { "screen_name": { "content": "CoolPerson2000" } } },
-    "remove_from_followers_warn": { "message": "TO USUNIE ICH Z LISTY OBSERWUJE, NIE Z TWOJEJ LISTY OBSERWOWANYCH KONT." },
+    "remove_from_followers_warn": { "message": "TO USUNIE TĄ OSOBĘ Z TWOICH OBSERWUJĄCYCH, NIE Z LISTY OSÓB, KTÓRE OBSERWUJESZ." },
     "remove_from_followers_button": { "message": "Zaprzestań ich obserwacji" },
-    "able_in_future": { "message": "Będzie to konto mogło cię zaobserwować ponownie." },
+    "able_in_future": { "message": To konto będzie mogło zaobserwować cię ponownie." },
```
Comment (dimdenGD) [inline _locales/pl/messages.json:133]: Missed quote

File: _locales/pl/messages.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "help": { "message": "Pomoc", "description": "noun" },
     "terms": { "message": "Warunki", "description": "Shortest word that you can find for Terms of service" },
     "privacy": { "message": "Prywatność", "description": "Shortest word that you can find for Privacy policy" },
-    "about": { "message": "Info" },
+    "about": { "message": "O" },
```
Comment (dimdenGD) [inline _locales/pl/messages.json:21]: This is too short and would be confusing to users

Comment (lesyeuxdechris) [reply _locales/pl/messages.json:21]: In Polish for "About us" or "About Twitter" (example), we usually use "O nas" or "O Twitterze".

Example from Google:
image

File: _locales/pl/messages.json (hunk 1 above)
Comment (lesyeuxdechris) [reply _locales/pl/messages.json:133]: omw to fix this, didn't notice it. Sorry!

File: _locales/pl/messages.json [hunk 3]
```diff
@@ -128,9 +128,9 @@
     "copy_profile_link": { "message": "Kopiuj link do profilu" },
     "remove_from_followers": { "message": "Usuń z obserwujących", "description": "This button unfollows person from you, not unfollows you from them" },
     "remove_from_followers_sure": { "message": "Czy na pewno chcesz usunąć @$SCREEN_NAME$ z twojej listy obserwujących?", "placeholders": { "screen_name": { "content": "CoolPerson2000" } } },
-    "remove_from_followers_warn": { "message": "TO USUNIE ICH Z LISTY OBSERWUJE, NIE Z TWOJEJ LISTY OBSERWOWANYCH KONT." },
+    "remove_from_followers_warn": { "message": "TO USUNIE TĄ OSOBĘ Z TWOICH OBSERWUJĄCYCH, NIE Z LISTY OSÓB, KTÓRE OBSERWUJESZ." },
```
Comment (wermipls) [inline _locales/pl/messages.json:131]: **tę** osob**ę**

**Pull request closed without merging**